nix/README.md

```diff
@@ -111,7 +111,5 @@ As another example, consider the file `__std__/cells/plutus/library/default.nix`
 
 Migration to `std` will be 100% complete once all these are done:
```

Comment:
Do you plan to do all these here, or are these still todos?

Reply:
I'll spread those out into multiple PRs. 
Maybe I've learned to keep my PRs short :P